README.md

```diff
@@ -1 +1,17 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## This is an autobiography from Yinsu Han for HW01.
+My name is **Yinsu Han**. I am a rising sophomore in the college, planning to major in **mathematics**. I am always excited to learn new things. Here are several things I enjoy apart from learning:
+
+* *Playing basketball*
+* *Listening to music/Going to concerts*
+* *Hanging out with my friends*
+
+### This is a picture of mine in downtown Chicago
+![picture](https://github.com/YinsuH/hw01/blob/master/dt_chicago.JPG)
+
+
+#### This includes the reference information to finish HW01.
+To complete the assignment, I first watched the video demonstration from https://cfss.uchicago.edu/homework/edit-readme/ and I find it really helpful to get the overall idea. Next, I referenced the mastering-markdown website(https://guides.github.com/features/mastering-markdown/) from github to know the basic syntax required to complete this homework.
```

Comment:
Don't forget the `[]()` syntax for embedding URLs in text. It makes the document more readable. For instance, [I first watched the video demonstration](https://cfss.uchicago.edu/homework/edit-readme/)

Reply:
I agree that integrating the links into your texts makes for a slightly cleaner looks.

Reply:
Thank you very much for that! I didn't know how exactly we could do that before!